### PR TITLE
Subscriptions > Personal info Removal

### PR DESCRIPTION
**Pull request by afterxleep**
Task/Issue URL: https://app.asana.com/0/72649045549333/1205043809052693/f

**Description**:
- Adds links to MacOS and Windows browser dowload from Personal Information renewal
- Removes all waitlist functionality related to macOS and Windows browser. (Use a simple SwiftUI View instead as there's no waitlist anymore)

**Steps to test this PR**:

**⚠️  Requires building the alpha app and a sandbox account**

1.  Get a privacy pro subscription 
2. Go to Personal Information Removal
3. Test blue buttons
4. Test the Share buttons in the "DuckDuckGo for Mac" and "DuckDuckGo for Windows" work
5. Test that tapping "Looking for the Windows version?" replaces view contents with the Windows donwload info.
6. Go to Settings > "DuckDuckGo App for Mac" and "DuckDuckgo App for Windows" and make sure both links and views work.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by afterxleep** [comment]
Hey @samsymons .. FYI This PR reviews the waitlist functionality for macOS and Windows Browsers as mentioned in [Asana.](https://app.asana.com/0/414709148257752/1206564801242964/f)

File: DuckDuckGo/SettingsGeneralView.swift [hunk 1]
```diff
@@ -37,6 +37,13 @@ struct SettingsGeneralView: View {
             NavigationLink(destination: WidgetEducationView()) {
                 SettingsCellView(label: UserText.settingsAddWidget)
             }
+            
+            if #available(iOS 15.0, *) {
+                NavigationLink(destination: SubscriptionPIRView()) {
+                    SettingsCellView(label: UserText.settingsAddWidget)
+                }
+            }
+
```
Comment (miasma13) [inline DuckDuckGo/SettingsGeneralView.swift]: This is probably some leftover from testing.

File: DuckDuckGo/SettingsViewModel.swift [hunk 2]
```diff
@@ -457,8 +457,8 @@ extension SettingsViewModel {
         case .fireproofSites: pushViewController(legacyViewProvider.fireproofSites)
         case .autoclearData: pushViewController(legacyViewProvider.autoclearData)
         case .keyboard: pushViewController(legacyViewProvider.keyboard)
-        case .windowsApp: pushViewController(legacyViewProvider.windows)
-        case .macApp: pushViewController(legacyViewProvider.mac)
+        // case .windowsApp: pushViewController(legacyViewProvider.windows)
+        // case .macApp: pushViewController(legacyViewProvider.mac)
```
Comment (miasma13) [inline DuckDuckGo/SettingsViewModel.swift]: Can be removed?

File: DuckDuckGo/DesktopDownloadViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    init(platform: DesktopDownloadPlatform) {
+        self.platform = platform
+        self.browserDetails = .init(platform: platform)
+    }
+    
+    func copyLink() {
+        UIPasteboard.general.url = downloadURL
+    }
+    
+    func switchPlatform() {
```
Comment (miasma13) [inline DuckDuckGo/DesktopDownloadViewModel.swift:45]: When doing the switch by tapping the "Looking for..." all items change appropriately except the URL link which stays the same (even though it copies the updated URL).

**Review by miasma13 — APPROVED** [review]

File: DuckDuckGo/SettingsGeneralView.swift (hunk 1 above)
Comment (afterxleep) [reply DuckDuckGo/SettingsGeneralView.swift]: OOF, yea.  😅,. thanks!

**Pull request merged**